Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -244,10 +244,23 @@ private void RunExpirationChecks()
   {
     var senders = Store.GetSenders();
     string[] objectIdsList = ChangedObjectIds.Keys.ToArray();
```

Comment:
Why this can't return UniqueId already?

Reply:
I mean why we don't  store unique ids already

Reply:
OK. I got it. API returns element ids from events

Reply:
:D Yes